packages/video_player_avplay/tizen/src/video_player.cc

```diff
@@ -163,6 +163,16 @@ void VideoPlayer::SendPlayCompleted() {
   PushEvent(flutter::EncodableValue(result));
 }
 
+void VideoPlayer::SendIsPlayingState(bool is_playing) {
+  flutter::EncodableMap result = {
+      {flutter::EncodableValue("event"),
+       flutter::EncodableValue("isPlayingStateUpdate")},
+      {flutter::EncodableValue("isPlayingState"),
```

Comment:
For reduce confusion with other platforms, I recommend to use the same key name.
(`isPlayingState` -> `isPlaying`)
https://github.com/flutter/packages/pull/3261/files#diff-7bf3fe00ba4225d8c58e9e7525747626af4d510cf2faf3986b41374ee238bcfaR240